src/Symfony/Bridge/Twig/Tests/Extension/AbstractBootstrap3LayoutTest.php

```diff
@@ -1863,6 +1863,22 @@ public function testInteger()
         );
     }
 
+    public function testIntegerTypeWithGroupingRendersAsTextInput()
```

Comment:
those changes should not be here, right? :blush: 

Only `SwitchUserListener.php` and `SwitchUserListenerTest.php` should be changed